src/site/stages/build/page-builder/filters.js

```diff
@@ -97,10 +42,10 @@ function getFilterType(contentModelType) {
 function getFilteredEntity(contentModelType, entity) {
   // TODO: Filter properties based on content model type
   const entityTypeFilter = getFilterType(contentModelType);
-  const entityFilter = new Set([...blackList, ...entityTypeFilter]);
+  const entityFilter = new Set([...whitelists.global, ...entityTypeFilter]);
```

Comment:
I think it makes sense for now to spread both lists together, but we may find that some entities are deliberately ignoring some fields in our global list.

Reply:
Yeah, if we encounter that case it may require some more complexity in order to avoid blacklists while still effectively having a union operator.  We could:
- remove that property from the global whitelist and insert it into all of the other arrays where we want it, OR
- Come up with some sort of signal that indicates that a value should be removed from the union, like `['include_me', 'me_too', '-not_this']`

Reply:
Yup! Those are both good options. I'd lean toward the former for speed and explicitness, but don't have strong opinions at this point. If we happen to whitelist a few extra properties on some stray content models, it's probably not the end of the world. This can definitely be done as an enhancement later.